tutorials/simulators/1_aer_provider.ipynb

```diff
@@ -570,7 +570,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.7"
+   "version": "3.6.10"
```

Comment:
Not entirely sure this _should _ happen. Does the downgrading change anything?

Reply:
This is interesting.. The version is actually 3.7.7 in [my branch](https://github.com/cyx617/qiskit-tutorials/blob/master/tutorials/simulators/1_aer_provider.ipynb).  But it downgrades to 3.6.10 after I made the pull request.

Reply:
It seems like that the system information (e.g. python version) on the [qiskit tutorial page](https://qiskit.org/documentation/tutorials/simulators/1_aer_provider.html)  would be automatically updated according to the current system after the notebook is deployed from github. For example, the python version on the [github page](https://github.com/Qiskit/qiskit-tutorials/blob/master/tutorials/simulators/1_aer_provider.ipynb) is 3.7.7 but it's displayed as 3.8.5 after the deployment. I'm not sure if this is the case.